lego/apps/events/serializers/events.py

```diff
@@ -64,6 +65,9 @@ class EventReadDetailedSerializer(TagSerializerMixin, BasisModelSerializer):
     comment_target = CharField(read_only=True)
     cover = ImageField(required=False, options={'height': 500})
     company = CompanyField(queryset=Company.objects.all())
+    responsible_group = AbakusGroupField(
+        queryset=AbakusGroup.objects.all(), required=False, allow_null=True
```

Comment:
think we should change this to `required=True` in the serializer eventually

Reply:
yes, only required in the webapp for right now